src/utils/senOppfolgingUtils.ts

```diff
@@ -0,0 +1,17 @@
+import { SenOppfolgingKandidatResponseDTO } from "@/data/senoppfolging/senOppfolgingTypes";
+import { addDays, dagerMellomDatoerUtenAbs } from "@/utils/datoUtils";
+
+export const isVarselUbesvart = (
```

Comment:
Skal vi kjøre helt enkel `function` her? 😊
```suggestion
export function isVarselUbesvart(
```

Reply:
Kan godt det!